### PR TITLE
XX 149 | SFRadioGroup

### PR DESCRIPTION
**Pull request by Yopo-Nicolas**

### CODE REVIEW

File: src/Components/SFRadioGroup/SFRadio/SFRadio.stories.tsx [hunk 1]
```diff
@@ -50,59 +53,9 @@ export default {
...
-    TouchRippleProps: {
-      table: {
-        disable: true
-      }
     }
   }
-} as Meta;
+} as ComponentMeta<typeof SFRadio>;
 
 const Template: Story<SFRadioProps> = (args) => <SFRadio {...args} />;
```
Comment (pdcastano) [inline src/Components/SFRadioGroup/SFRadio/SFRadio.stories.tsx:60]: Update to ComponentStory according to the new Storybook Docs

**Review by pdcastano — CHANGES_REQUESTED** [review]
- Check component props order.

File: .storybook/main.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ module.exports = {
     '../src/Components/SFIconButton/*.stories.@(js|jsx|ts|tsx)',
     '../src/Components/SFLink/*.stories.@(js|jsx|ts|tsx)',
     '../src/Components/SFPaper/*.stories.@(js|jsx|ts|tsx)',
+    '../src/Components/SFSwitch/*.stories.@(js|jsx|ts|tsx)',
+    '../src/Components/SFRadioGroup/SFRadio/*.stories.@(js|jsx|ts|tsx)',
+    '../src/Components/SFRadioGroup/*.stories.@(js|jsx|ts|tsx)',
```
Comment (pdcastano) [inline .storybook/main.js:17]: this should be in alphabetic order

**Review by pdcastano — CHANGES_REQUESTED** [review]

File: .storybook/main.js (hunk 2 above)
Comment (Yopo-Nicolas) [reply .storybook/main.js]: Done

**Comment by pdcastano** [comment]
LGTM

**Review by pdcastano — APPROVED** [review]

**Pull request merged**